scio-avro/src/main/scala/com/spotify/scio/avro/syntax/SCollectionSyntax.scala

```diff
@@ -203,7 +205,73 @@ final class ProtobufSCollectionOps[T <: Message](private val self: SCollection[T
       shardNameTemplate,
       tempDirectory
     )
-    self.write(ProtobufIO[T](path))(param)
+    self.write(ProtobufObjectFileIO[T](path))(param)
+  }
+}
+
+final class TypedMagnolifyProtobufSCollectionOps[T](private val self: SCollection[T])
+    extends AnyVal {
+
+  /**
+   * Save this SCollection as a Protobuf file.
+   *
+   * Protobuf messages are serialized into `Array[Byte]` and stored in Avro files to leverage Avro's
+   * block file format.
+   */
+  def saveAsProtobufFile[U <: Message](
```

Comment:
```suggestion
  def saveAsProtobufFile[U <: Message: ClassTag](
```

just a bit neater than having to write out the `implicit ct...` param IMO